yarn.lock

```diff
@@ -3272,7 +3272,10 @@ cartobox-promises-utility@nycplanning/cartobox-promises-utility#v1.0.2:
   resolved "https://codeload.github.com/nycplanning/cartobox-promises-utility/tar.gz/57f1c96a0bf03c47e048c30d202643cc49db3c69"
   dependencies:
     ember-cli-babel "^6.6.0"
+<<<<<<< HEAD
     ember-fetch "5.0.0"
+=======
+>>>>>>> bfad4be8a1452cb6895bfdf0cae8a109859af731
```

Comment:
Eeek! Looks like some merge conflict markup got in here

Reply:
Ahh! Will fix